engine/class_modules/sc_shaman.cpp

```diff
@@ -4644,7 +4552,6 @@ struct flame_shock_t : public shaman_spell_t
 {
   flame_shock_spreader_t* spreader;
   const spell_data_t* elemental_resource;
-  const spell_data_t* t20_4pc_bonus;
```

Comment:
This got caught looking for unused variables. Didn't feel necessary to split out into its own change given its just the one line.